zaza/openstack/charm_tests/ceph/tests.py

```diff
@@ -1100,6 +1127,49 @@ def test_004_migration_and_multisite_failover(self):
         # Verify Data Integrity.
         self.assertEqual(secondary_content, primary_content)
 
+        # Scaledown and verify replication has stopped.
+        logging.info('Checking multisite scaledown')
+        zaza_model.remove_relation(
+            self.primary_rgw_app,
+            self.primary_rgw_app + ":master",
+            self.secondary_rgw_app + ":slave"
+        )
+
+        # wait for sync stop
+        self.wait_for_scaledown(self.primary_rgw_app)
+        self.wait_for_scaledown(self.secondary_rgw_app)
+
+        # Refresh client and verify objects are not replicating.
+        primary_client = boto3.resource("s3",
+                                        verify=False,
+                                        endpoint_url=primary_endpoint,
+                                        aws_access_key_id=access_key,
+                                        aws_secret_access_key=secret_key)
+        secondary_client = boto3.resource("s3",
+                                          verify=False,
+                                          endpoint_url=secondary_endpoint,
+                                          aws_access_key_id=access_key,
+                                          aws_secret_access_key=secret_key)
+
+        # IO Test
+        container = 'scaledown-container'
+        test_data = 'Scaledown Test data'
+        secondary_client.Bucket(container).create()
+        secondary_object = secondary_client.Object(container, 'scaledown')
+        secondary_object.put(
+            Body=test_data
+        )
+
```

Comment:
Using something like this may be cleaner than the following try/except block:

```
with self.assertRaises(botocore.exceptions.ConnectionError):
    primary_content = self.fetch_rgw_object(
        primary_client, container, 'scaledown'
    )
```